### ISSUE TITLE
New onboarding process

### ISSUE THREAD

**Issue by kaloudis** [ZeusLN/zeus#2342]
Currently our onboard process is quite lacking. Overview of the app is minimalistic. You can choose Settings before getting started but users aren't walked through them.

Let's ask users:
- Remote connection or not
- If they want fiat rates enabled
- If they want to apply a password/pin
- If they want to enable clipboard reads
- [future] if using embedded node, whether to use more private Neutrino, or more reliable alternative connection (esplora/electrum)

**Comment by newtonick** [comment]
I agree a more guided setup/onboarding flow would be nice. Even as an advanced user when I need to choose between "Quick Start" vs "Advanced Set-up" it's not clear what will happen next when I click either option. I think the main flow should lead the user to use the embedded node unless they choose a path/option to connect to a remote node (ie umbrel/startos/etc).
